packages/rich-text/src/component/index.native.js

```diff
@@ -827,7 +827,15 @@ export class RichText extends Component {
 				: this.state.width;
 
 		return (
-			<View>
+			<View
+				style={ [
```

Comment:
I think you can remove the array from here since you add only one object inside :)

Reply:
Sounds good! Updated 👍